### ISSUE TITLE
#2581 + issue of custom columns in listview

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#2625]
#### Issue

On videoscreen you will see the this issues: http://www.screencast.com/t/qZyHPV9ii

1. #2581
2. When you create columns via Layout options, after clear button & search it disapears
3. Multi select area is hard visible & usable + its very hard to work with it if you want choose more columns, can we use someting like this: http://isteven.github.io/angular-multi-select/#/demo-minimum

#### Your Environment

* SuiteCRM Version used: 7.7.7. Official demo
* Browser name and version: Chrome 54.x 64bit
* Operating System and version: Win 10 Home

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [#2581 + issue of custom columns in listview](https://trello.com/c/q2CXlDow/52-2581-issue-of-custom-columns-in-listview)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
